packages/components/controlled-input/README.mdx

```diff
@@ -0,0 +1,3 @@
+# @contentful/f36-controlled-input
+
+This package is part of [forma-36](https://github.com/contentful/forma-36). See the repo for more details.
```

Comment:
Looking good! Thx a lot for the contribution! 🥇 
I would encourage you to write documentation for the component if that's not too much trouble. (we could to it in the future, but it will be a bit painful later I think 😉)
For the README we have a following structure (sometimes some of the sections are not needed depends of the component):

## Table of contents

- [How to use ControlledInput](#how-to-use-controlled-input)
- [Component variations](#component-variations)
- [Code examples](#code-examples)
- [Content recommendations](#content-recommendations)
- [Accessibility](#accessibility)

Reply:
I'll prepare two follow-up PRs which will restructure this folder - it will be f36-inputs containing multiple components. If Should I create this for every component and for the containing package? If so, I would like to file this in another PR (but I'll do it definitively) :)

Reply:
sure thing! thanks a lot! ♥️

Reply:
My pleasure and thank you :)